api/communication-api/src/main/java/com/cognifide/aet/communication/api/metadata/Test.java

```diff
@@ -34,6 +35,10 @@
 
   private final String preferredBrowserId;
 
+  private boolean isRerunned = false;
```

Comment:
You don't have to initialize, `false` is default value for `boolean`.

Reply:
Done https://github.com/Cognifide/aet/pull/337/commits/ce375139f8eca1a90e313beb3ecb146549373027